src/vivarium_census_prl_synth_pop/tools/cli.py

```diff
@@ -174,9 +165,10 @@ def make_results(
     peak_memory: int,
     max_runtime: str,
 ) -> None:
-    """Create final results datasets from the raw results output by observers
-
-    :param output_dir: Directory where the raw results are stored
+    """Create final results datasets from the raw results found in the OUTPUT_DIR
+    and store them in a time-stamped and versioned directory structure using LABEL_VERSION.
+    \f
```

Comment:
`\f` breaks the click --help docstring. So when you do --help it won't show the `:param` stuff below it since those are duplicates of what's in the actualy click.option help string. (but the param stuff is needed for sphinx documentation so it'll show up there)